### PR TITLE
EM-850: Updated Nav Breakpoints For Top Nav User Icon

### PR DESCRIPTION
**Pull request by MKwenhua**
post jobs and find talent careerbuilder for employers 2017-01-25 11-37-12

I change the top-nav break-points to account for the extra space the new user account widget takes up.

### CODE REVIEW

**Comment by arelia** [comment]
Looks good! Make sure to point out this change to POs when you move this story to Done. They may want to reduce the space between nav items to allow a lower breakpoint between full nav and mid-range nav

File: sass/directives/_flyout.scss [hunk 1]
```diff
@@ -1,26 +1,55 @@
...
-  display: none;
-
-  @media only screen and (max-width: $mobile-nav-max) {
-    position: relative;
-    top: 0;
-    left: 0;
-    border: 0;
-    border-radius: 0;
-    padding: 0;
+@mixin nav_flyout($top, $left, $offset: 25%) {
```
Comment (arelia) [inline sass/directives/_flyout.scss]: 👍

Comment (MKwenhua) [inline sass/directives/_flyout.scss]: should I merge?

**Comment by arelia** [comment]
@MKwenhua Don't merge this PR until you're ready to merge [Employer PR #636](https://github.com/cbdr/employer/pull/636). It looks like there's one change requested from @ElliottAYoung and you may have already made the changes for @toastercup although it's indicating he's requested changes

**Comment by MKwenhua** [comment]
@arelia I think the remaining change @ElliottAYoung requested was that I remove the branch in *bower.json* that points to `employer-style-base` back after this is resolved.

**Pull request merged**